runtime/tr.source/trj9/optimizer/UnsafeFastPath.cpp

```diff
@@ -149,6 +149,16 @@ int32_t TR_UnsafeFastPath::perform()
          bool isVolatile = false;
          bool isArrayOperation = false;
          bool isByIndex = false;
+         int32_t objectChild = 1;
+         int32_t offsetChild = 2;
```

Comment:
Outside the scope of this PR but this does beg the question as to why JITHelpers class (https://github.com/eclipse/openj9/blob/9cd50e6df6ab349979e94978751b9f28818e0fb8/jcl/src/java.base/share/classes/com/ibm/jit/JITHelpers.java) is not static class (i.e. static in the sense that it doesn't seem to have instance fields and thus the methods can be static). Thoughts?

Reply:
The class is not static as a means of reducing the ability to access it - for the same reasons Unsafe is not static - the singleton accessor has a security check.